lab_3/main.py

```diff
@@ -123,19 +123,56 @@ def predict_next_sentence(self, prefix: tuple) -> list:
             counter -= 1
         return prefix_list
 
-
-
-def encode(storage_instance, corpus) -> list:
-    new_corpus = []
-    id_sentence = []
-    for sentence in corpus:
-        for word in sentence:
-            number = storage_instance.put(word)
-            id_sentence.append(number)
-        new_corpus.append(id_sentence)
+    def encode(storage_instance, corpus) -> list:
```

Comment:
@mariia1299 
enode and split_by_sentence should not be members of NGramTrie class